core/templates/dev/head/components/summary_tile/question_summary_tile_directive.html

```diff
@@ -0,0 +1,73 @@
+<md-card class="oppia-activity-summary-tile" style="z-index: 5; width: 100%;">
+  <a ng-href="<[getQuestionLink()]>" style="text-decoration: none;">
+    <div class="oppia-question-summary-tile" style="border-left-color: <[getQuestionStripColor()]>">
+      <div class="oppia-question-description oppia-activity-summary-contents">
+        <em>
+          <b>Question: </b>
+          <span>
+            <[getQuestionDescription()]>
+          </span>
+        </em>
+      </div>
+      <span ng-if="getLastUpdatedDatetime()" class="oppia-question-last-updated">
+        <span class="fa fa-clock-o fa-lg oppia-dashboard-clock-icon" uib-tooltip="Last updated" tooltip-placement="top">
+          <[getLastUpdatedDatetime()]>
+            <span class="oppia-icon-accessibility-label">Last Updated</span>
+        </span>
+        <span ng-if="!getLastUpdatedDatetime()" translate="I18N_LIBRARY_N/A" class="oppia-inactive-summary-tile-attribute">
+        </span>
+      </span>
+      <br>
+      <div class="oppia-dashboard-tagged-skills oppia-activity-summary-contents">
+        <span ng-repeat="skill in getTaggedSkills">
+          <span class="oppia-dasboard-skill">
+            <[skill]>
+          </span>
+        </span>
+      </div>
+      <div class="oppia-dashboard-tagged-skills oppia-activity-summary-contents" ng-if="!getTaggedSkills">
+        <span class="oppia-dasboard-skill">
+          <em>Edit Question to tag skills.</em>
+        </span>
+      </div>
+    </div>
+    <div ng-if="!isPlaylistTile()" class="title-section-mask" ng-mouseenter="setHoverState(true)" ng-mouseleave="setHoverState(false)"></div>
+  </a>
+</md-card>
+
+<style>
+  .oppia-question-summary-tile .oppia-activity-summary-contents {
```

Comment:
Since this won't be needed in other html files, write the html tag name before all these styles to confine it to this directive, which I think is question-summary-tile?
or have .oppia-question-summary-tile in front of all styles.